lang/src/org/partiql/lang/eval/builtins/FromUnixTimeFunction.kt

```diff
@@ -0,0 +1,34 @@
+package org.partiql.lang.eval.builtins
+
+import com.amazon.ion.Timestamp
+import org.partiql.lang.eval.Environment
+import org.partiql.lang.eval.ExprValue
+import org.partiql.lang.eval.ExprValueFactory
+import org.partiql.lang.eval.NullPropagatingExprFunction
+import org.partiql.lang.eval.bigDecimalValue
+import java.math.BigDecimal
+
+/**
+ * Builtin function to convert the given unix epoch into a PartiQL `TIMESTAMP` [ExprValue]. A unix epoch represents
+ * the seconds since '1970-01-01 00:00:00' UTC. Largely based off MySQL's FROM_UNIXTIME.
+ *
+ * Syntax: `FROM_UNIXTIME(unix_timestamp)`
+ * Where unix_timestamp is a non-negative (potentially decimal) numeric value.
```

Comment:
This seems to no longer be true, we can pass a negative numeric value. 
I like that you added the syntax in the comment. I think it would be nice to also add the return type or at least a description of the expected function's return values

Reply:
Good catch, thanks! Also edited to make it more clear on the expected return values.